### ISSUE TITLE
Add an "outIngredients" in recipe format

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/minecraft-data#10]
http://minecraft.gamepedia.com/Written_Book is an unordered recipe, ie it uses ingredients. But it doesn't consume the written book so there should be a "outIngredients" in the recipe format.

**Comment by Gjum** [comment]
Good catch. This should totally be added, as it should not break any present implementations and is trivial to add.
